src/client/components/app/Transactions/WithdrawCreditTx.tsx

```diff
@@ -37,12 +37,19 @@ export const WithdrawCreditTx: FC<BorrowCreditProps> = (props) => {
   const [transactionLoading, setLoading] = useState(false);
   const [targetAmount, setTargetAmount] = useState('1');
   const [errors, setErrors] = useState<string[]>(['']);
-  const selectedCredit = useAppSelector(LinesSelectors.selectSelectedLine);
+  // const selectedCredit = useAppSelector(LinesSelectors.selectSelectedLine);
```

Comment:
Instead of refactoring all the components to update logic, it's probs easier to update the `selectSelectedLine` selector to first check `selectedLine` and then check `selectedPosition.line` to pull data from. No updates to components required at all

Reply:
great idea! fixed